src/Shared/UnitTests/TypeLoader_Tests.cs

```diff
@@ -20,6 +21,8 @@ public class TypeLoader_Tests
         private static readonly string ProjectFileFolder = Path.Combine(BuildEnvironmentHelper.Instance.CurrentMSBuildToolsDirectory, "PortableTask");
         private const string ProjectFileName = "portableTaskTest.proj";
         private const string DLLFileName = "PortableTask.dll";
+        private static string PortableTaskFolderPath = Path.GetFullPath(
```

Comment:
Can you use this in the other spot, or use that one here?

Reply:
I do think that would be nice, but they're in different test assemblies, and I don't think it makes sense to have this test there or that test here. In other words, the two test classes have no knowledge of each other, and that seems appropriate.